platformio.ini

```diff
@@ -137,7 +137,9 @@ build_flags =
   -DWLED_USE_MY_CONFIG
   ; -D USERMOD_SENSORSTOMQTT
   #For ADS1115 sensor uncomment following
-  ; -D USERMOD_ADS1115
+  ;-D USERMOD_ADS1115
```

Comment:
Please revert anything in `platformio.ini`

Reply:
This still needs addressing. Please remove `platformio.ini` from the PR.
It would be best to also rebase to 0_15 branch.

Reply:
@blazoncek sorry what do you mean? Do you want me to fork the 0_15 branch and modify that repo instead of the main one?

Reply:
I would suggest you follow [this](https://github.com/Aircoookie/WLED/wiki/How-to-properly-submit-a-PR) Wiki article and use 0_15 branch as your base.

In essence:
```
git fetch
git checkout -b 0_15 upstream/0_15 --no-track
git pull
git checkout -b strip-timer
<edit changes>
git commit -a -m "comment"
git push origin
```
And then make a new PR from your "strip-timer" branch.

If you do not mind *polluting* your repository (as you made changes in "main" branch) then just pull `upstream/0_15` and change destination branch of this PR.